### PR TITLE
Document the assessment report

### LINKED ISSUE #748: Add assessment finding identifier and reference link for each item found in the assessment
The assessment summary (and details ideally) should reference an assessment finding identifier, and a hyper link back to a README heading that explains the finding and recommended resolution path(s).

e.g.
### AF001 - DBR Version doesn't support Unity Catalog features
Resolution: Upgrade cluster/job runtime to a supported version (recommend DBR 13.3LTS is higher

### PR DESCRIPTION
**Pull request by dmoore247**
## Changes
Documenting the Assessment (main) report. This is the initial revision of the document to help guide users through interpreting the Assessment report. This document is fairly complete with inline screen shots, document introduction, Assessment Finding Index.

Assessment queries and messages may be numbered and linked in future PRs but not this PR.

### Linked issues

Resolves #748 

### Functionality 

- [x] added relevant user documentation

### Tests

- [x] manually tested (reviewed)
Circulating draft with field eng and select customers

### CODE REVIEW

File: docs/assessment.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# Assessment Report Summary
+
+# Assessment Dashlets
```
Comment (nfx) [inline docs/assessment.md:3]:
```suggestion
# Assessment Widgets
```

that's how we're calling them. Don't introduce new term

File: docs/assessment.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+## Incompatible Global Init Scripts
+These are Global Init Scripts that are incompatible with Unity Catalog compute. As a reminder, global init scripts need to be on secure storage (Volumes or a Cloud Storage account and not DBFS)
+
+
+# Assessment Finding Index
+
+This page will help explain UCX Assessment findings with a recommended action
+
+### AF001 - DBR ...
```
Comment (nfx) [inline docs/assessment.md:37]: this doesn't work like this now

**Review by nfx — CHANGES_REQUESTED** [review]

File: docs/assessment.md (hunk 2 above)
Comment (dmoore247) [reply docs/assessment.md]: This was an early draft

**Comment by dmoore247** [comment]
@nfx Ready for review and merge.

File: docs/assessment.md [hunk 3]
```diff
@@ -0,0 +1,188 @@
+# UCX Assessment Introduction
+This document describes the Assessment Report generated from the UCX tools. The main assessment report includes dashlets, widgets and details of the assessment findings and common recommendations made based on the Assessment Finding (AF) Index entry.
+
+# Assessment Report Summary
+The Assessment Report (Main) is the output of the Databricks Labs UCX assessment workflow. This report queries the $inventory database (e.g. `ucx`) and summarizes the findings of the assessment. The link to the Assessment Report (Main) can be found in your home folder, under `.ucx` in the README.py file. The user may also directly navigate to the Assessment report by clicking on `Dashboards` icon on the left to find the Dashboard.
+
+# Assessment Dashlets
```
Comment (nfx) [inline docs/assessment.md:7]: I repeat, DO NOT USE IRRELEVANT TERMS.
```suggestion
# Assessment Widgets
```

File: docs/assessment.md [hunk 4]
```diff
@@ -0,0 +1,188 @@
...
+<img width="1655" alt="image" src="https://github.com/databrickslabs/ucx/assets/1122251/808f7c68-fcc7-4caa-bab2-03f49a382256">
```
Comment (nfx) [inline docs/assessment.md:8]: Images are not version controllable, try to avoid them. We're changing to lakeview soon, anyway

**Review by nfx — CHANGES_REQUESTED** [review]

File: docs/assessment.md (hunk 4 above)
Comment (dmoore247) [reply docs/assessment.md:8]: Please take a look at the doc, the images really pull it together for the reader, breaking the text into bite size bits.

As far as versioning images go, the images are immutable and not intended to be versioned in place, github does a really nice job. When Lakeview comes along it's just 5 copy paste operations.

Comment (nfx) [reply docs/assessment.md:8]: image

Here's what I'm talking about.

Besides, we're changing the dashboards to lakeview soon

**Pull request merged**